### PR TITLE
[es] add spanish translation for Agile software Development

### LINKED ISSUE #562: [Spanish] localize 'Agile Software Development'
Localize the `content/en/agile_software_development.md` document to Spanish

### PR DESCRIPTION
**Pull request by anyulled**
Resolves #562

### CODE REVIEW

**Comment by raelga** [comment]
@anyulled Do the commits a8cddf74dc240712118a8d7bd9e3c679238ca7ac and 0f1be16ad71f53f3f49d122e57f2fa8adc93f3f5 belong to you PR?

**Comment by anyulled** [comment]
Hi, @raelga, no, and I was about to ask from which branch do I need to work from there's a dev-es and dev-es-rebase and I'd like which one should I use.

**Comment by electrocucaracha** [comment]
> Hi, @raelga, no, and I was about to ask from which branch I need to work as there's a dev-es and dev-es-rebase, and I'd like to know which one I should use.

@anyulled it's `dev-es` the another will be removed soon.

**Comment by anyulled** [comment]
@raelga @electrocucaracha fixed!

**Review by raelga — APPROVED** [review]

**Comment by electrocucaracha** [comment]
@anyulled can you add `Agile` and `Development` into the [wordlist](https://github.com/cncf/glossary/blob/dev-es/content/es/.wordlist.txt) to pass the CI spellchecker?

File: content/es/agile_software_development.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: Agile Software Development
```
Comment (electrocucaracha) [inline content/es/agile_software_development.md:2]:
```suggestion
title: Desarrollo ágil de software
```

**Comment by anyulled** [comment]
> @anyulled can you add `Agile` and `Development` into the [wordlist](https://github.com/cncf/glossary/blob/dev-es/content/es/.wordlist.txt) to pass the CI spellchecker?

Done

**Review by electrocucaracha — APPROVED** [review]

**Pull request merged**